theano/tensor/basic.py

```diff
@@ -2338,6 +2338,12 @@ def conj(z):
 def complex_from_polar(abs, angle):
     """Return complex-valued tensor from polar coordinate specification."""
 
+
+@_scal_elemwise
+def trigamma(x):
```

Comment:
If you make m an input instead of a parameter, it would work. I'm not sure what the error with wrong value come from. It could be because you create the first scalar op with a props. Maybe it isn't supported somewhere.

Is there a reason why m was make a props instead of an inputs?